andrewinsoul/Fast-Food-Fast
# #160991406 User view index through link

#### What does this PR do?
This PR allows the index page to be accessed through a link

#### Description of Tasks to be completed
- Set response headers in the file that boots the server
- Modify test to fix failing build
- Disable eslint for the client folder

#### How should this be manually tested?
-  Clone repo and npm install all dependencies
- Start up the server with npm start and access the homepage
- You should see the index view of the application

#### What are the relevant Pivotal Tracker Stories?
[#160991406](https://www.pivotaltracker.com/n/projects/2196515)
